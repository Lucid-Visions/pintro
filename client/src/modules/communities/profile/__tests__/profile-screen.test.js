import React from 'react'
import renderer from 'react-test-renderer'
import ProfileScreen from '../components/profile-screen'

describe('<ProfileScreen />', () => {

  let wrapper = {}

  const jsx = renderer.create(<ProfileScreen />)

  describe('Snapshot test', () => {
    it('matches snapshot', () => {
      expect(jsx).toMatchSnapshot()
    })
  })

  describe('', () => {
    
  })
})