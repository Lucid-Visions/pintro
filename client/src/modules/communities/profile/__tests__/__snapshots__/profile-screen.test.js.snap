// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ProfileScreen /> Snapshot test matches snapshot 1`] = `
<RCTScrollView>
  <View>
    <View>
      <Text>
        TEST
      </Text>
      <Text>
        TEST
      </Text>
    </View>
  </View>
</RCTScrollView>
`;
